docs/getting-started/full-dotnet.rst

```diff
@@ -0,0 +1,102 @@
+.. include:: /stub-topic.txt
```

Comment:
Changed to a .. caution:: box (like the note boxes but orange). The actual warning is in global-header.rst so we can easily tweak it later for the whole site.